### PR TITLE
Spikes script added

### PR DESCRIPTION
**Pull request by TosaLipkiss**
checking collision and reload scene if collide

### CODE REVIEW

File: Assets/Scripts/ObstacleSpike.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class ObstacleSpike : MonoBehaviour
+{
+	private void Start(){
+		
+	}
```
Comment (MrJackey) [inline Assets/Scripts/ObstacleSpike.cs]: This start method could be removed as it does not do anything yet.

File: Assets/Scripts/ObstacleSpike.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class ObstacleSpike : MonoBehaviour
+{
```
Comment (MrJackey) [inline Assets/Scripts/ObstacleSpike.cs]: This bracket should be on the line above.

File: Assets/Scripts/ObstacleSpike.cs [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+	private void Start(){
+		
+	}
+
+	public void OnCollisionEnter2D(Collision2D collision){
+
+		if(collision.gameObject.tag == "Player"){
```
Comment (MrJackey) [inline Assets/Scripts/ObstacleSpike.cs]: Would look good with some spaces before the parenthesis and bracket. Same with brackets on functions

**Review by MrJackey — CHANGES_REQUESTED** [review]
Looking good, some minor formatting issues only.

**Review by smeas — APPROVED** [review]

**Review by MrJackey — APPROVED** [review]
Looking good

**Pull request merged**